generators/service-watson-personality-insights/templates/python/instrumentation.py

```diff
@@ -1,10 +1,23 @@
 from ibmcloudenv import IBMCloudEnv
 from watson_developer_cloud import PersonalityInsightsV3
 
-personality_insights = PersonalityInsightsV3(
-    username=IBMCloudEnv.getString('watson_personality_insights_username')['username'],
-    password=IBMCloudEnv.getString('watson_personality_insights_password')['password'],
-    version='2016-10-20')
-
+if IBMCloudEnv.getString('watson_personality_insights_apikey'):
+    iam_url = 'https://iam.stage1.bluemix.net/identity/token' if 'gateway-s.' in params.url else 'https://iam.bluemix.net/identity/token'
+    iam_apikey = api_key = IBMCloudEnv.getString('watson_personality_insights_apikey')
+    personality_insights = PersonalityInsightsV3(
+        url=IBMCloudEnv.getString('watson_personality_insights_url'),
+        iam_api_key=iam_apikey,
+        version='2017-10-13',
+        iam_url=iam_url)
+else:
+    personality_insights = PersonalityInsightsV3(
+        username=IBMCloudEnv.getString('watson_personality_insights_username')['username'],
```

Comment:
Why are we using `IBMCloudEnv.getString('watson_personality_insights_username')['username']` isntead of `IBMCloudEnv.getString('watson_personality_insights_username')` here